openscenario/openscenario_interpreter/src/syntax/performance.cpp

```diff
@@ -49,6 +51,7 @@ Performance::operator traffic_simulator_msgs::msg::Performance() const
   performance.max_deceleration = max_deceleration;
   performance.max_deceleration_rate = max_deceleration_rate;
   performance.max_speed = max_speed;
+  performance.max_jerk = max_jerk;
```

Comment:
maxJerk field does not exist in [Performance.](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/Performance.html)
Please remove it.